pkg/kubelet/container/runtime.go

```diff
@@ -97,6 +97,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Cadvisor FS Label
+	ImageLabel() string
```

Comment:
I'm not sure I like the use of the term `Label` here. I noticed that the possible values are `root | docker-images | rkt-images`. I think `FSType` or similar might be clearer, here and everywhere else. Second opinions welcome since this is a completely subjective thing.